roles/wisdom/wisdom_llm_load_test/templates/llm-load-test-config.json.j2

```diff
@@ -15,7 +15,7 @@
       "s3_separator": "/",
       "s3_base_path": "InferenceResults",
       "s3_component": "ModelMesh",
-      "s3_sub_component": "WatsonRuntime_v0.25.0",
+      "s3_sub_component": "WatsonRuntime_v0.27.0",
```

Comment:
I think it would be better to have the version identifier in the `config.yaml` file.
Try to understand how the `command_args.yaml` file works, and call:
```
./run_toolbox.py from_config widsdom ll_load_test
```
let me know if you don't follow how it works
(ignore the prefix/suffix parts)